setup.cfg

```diff
@@ -12,7 +12,7 @@ junit_suite_name = yarl_test_suite
 
 
 [flake8]
-ignore = E301,E302,E704,W503,W504,F811
+ignore = E203,E301,E302,E704,W503,W504,F811
```

Comment:
I'd rather inject a few `noqa`s if it's really necessary

Reply:
Too many `noqa`s should be injected. `blake` produces code incompatible with `flake8`.